d2l-step.js

```diff
@@ -31,37 +36,37 @@ class D2LStep extends mixinBehaviors([D2L.PolymerBehaviours.CustomBehaviours.Lan
 			.button-next {
 				margin-left: auto;
 			}
+		`;
+	}
 
-		</style>
-
-		<slot></slot>
-		<div class="footer">
-			<template is="dom-if" if="[[!hideRestartButton]]">
-				<d2l-button on-click="_restartClick">[[restartButtonTitle]]</d2l-button>
-			</template>
+	static async getLocalizeResources(langs) {
+		return getLocalizeResources(langs);
+	}
 
-			<d2l-button class="button-next" on-click="_nextClick" primary="" disabled$="[[disableNextButton]]">[[nextButtonTitle]]</d2l-button>
-		</div>
-`;
+	constructor() {
+		super();
+		this.hideRestartButton = false;
+		this.disableNextButton = false;
 	}
 
-	ready() {
-		super.ready();
-		if (!this.nextButtonTitle) {
-			this.nextButtonTitle = this.localize('stepper.defaults.next');
-		}
-		if (!this.restartButtonTitle) {
-			this.restartButtonTitle = this.localize('stepper.defaults.restart');
-		}
+	render() {
+		return html`
+			<slot></slot>
+			<div class="footer">
+				${this.hideRestartButton ? html`` : html`<d2l-button @click="${this._restartClick}">${!this.restartButtonTitle ? this.localize('stepper.defaults.restart') : this.restartButtonTitle}</d2l-button>`}
+
```

Comment:
Set slot attribte in both the `<d2l-button>` to `footer` for RTL functionality.

Reply:
Did you mean "We can set **class** attribute in both the ..."?
If so, this doesnt work since setting the class of 2 elements is different than having them in a wrapper and setting the class of that.
This is how it looks if I remove the div and add the footer class to each button
![image](https://user-images.githubusercontent.com/35306396/99297293-0ef10480-2816-11eb-8d12-65d3fa4a15f6.png)

Reply:
I mean adding `slot='footer' to the `<d2l-button>` 

```suggestion
${this.hideRestartButton ? html`` : html`<d2l-button slot='footer' @click="${this._restartClick}">${!this.restartButtonTitle ? this.localize('stepper.defaults.restart') : this.restartButtonTitle}</d2l-button>`}

<d2l-button  slot='footer'  class="button-next" @click="${this._nextClick}" primary ?disabled="${this.disableNextButton}">${!this.nextButtonTitle ? this.localize('stepper.defaults.next') : this.nextButtonTitle}</d2l-button>
```

Reply:
Looking at [this](https://lit-element.polymer-project.org/guide/templates#use-named-slots), it seems like the `slot` attribute is meant for if you have a component that has multiple slots, you can name them and then use the slot attribute to say which slot it should go into.